bundles/org.openhab.binding.dwdunwetter/src/main/resources/OH-INF/i18n/dwdunwetter_de.properties

```diff
@@ -11,7 +11,7 @@ thing-type.dwdunwetter.dwdwarnings.description = DWD Unwetterwarnungen für ein
 # thing types config
 
 thing-type.config.dwdunwetter.dwdwarnings.cellId.label = Cell-ID
-thing-type.config.dwdunwetter.dwdwarnings.cellId.description = ID der abzufragende Zelle. Siehe https\://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv. Es kann auch mittels % eine Gesamtmenge abgefragt werden, z.B. 8111% alle Gemeinden die mit 8111 anfangen.
+thing-type.config.dwdunwetter.dwdwarnings.cellId.description = ID der abzufragende Zelle. Siehe https\://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv, nur IDs die mit einer 8 (Ausnahme für Berlin: 7) beginnen werden unterstützt. Es kann auch mittels % eine Gesamtmenge abgefragt werden, z.B. 8111% alle Gemeinden die mit 8111 anfangen.
```

Comment:
Can you please remove this change ?
After the PR is merged, you can then update it in Crowdin.